### DISCUSSION TITLE
How to check if a property is overridden?

### DISCUSSION THREAD

**Discussion by RimuruChan** [jbevain/cecil#847] [Q&A]
![image](https://user-images.githubusercontent.com/42002296/167260824-eee46c41-5a34-45d7-881f-a5a4deaf0151.png)
Just like these two properties.
Thanks!

**Comment by Zastai** [comment]
If you mean, how do I tell from the `PropertyDefinition` that it had the `override` keyword in the source: you look at the accessor methods (such flags are stored separately for `get`/`set`/`put`/...).
And for a `MethodDefinition`, the logic is: a) it's virtual, b) it is not marked as a new slot, or it's flagged as having a covariant return type.
(See also [this code](https://github.com/Zastai/Zastai.Build.ApiReference/blob/69f785f0435c127562d3cd5e3e1cac8f4ba1f386/Zastai.Build.ApiReference/CSharpFormatter.cs#L10-L41).)

If you mean: how can I tell from the `PropertyDefinition` that there are derived types that have such an overriding property: you can't, unless you scan all types (and nested types) for derived classes and then match up their properties.
